johnsonandjohnson/Bodiless-JS
# chore(gatsby): gatsby-plugin-google-tagmanager version update to include fix on dataLayerName issue. (#60)

…address GTM issue.





## Changes

Update package gatsby-plugin-google-tagmanager version to address JS error issue.

## Test Instructions

1. Enable GTM via set GOOGLE_TAGMANAGER_ENABLED to "1" and proper GTM code for GOOGLE_TAGMANAGER_ID
in examples/test-site/.env.production

2. Serve site on http://localhost:9000/

Check if there are JS errors related to GTM feature.

## Related Issues

Fixes #60
